integration_tests/assets/docker-compose.documentation.override.yml

```diff
@@ -3,6 +3,7 @@ services:
   sync:
     depends_on:
       - plugind
+      - auth
       - rabbitmq
     environment:
-      TARGETS: "rabbitmq:5672 plugind:9503"
+      TARGETS: "rabbitmq:5672 plugind:9503 auth:9497"
```

Comment:
**Answer:** Well I had two solutions in mind: either replicate the old `BaseIntegrationTest` class with the authentication and call the new one `BaseDocumentationTest` and use it during the documentation integration test, or just add the authentication service (what I did; because it was more simpler I think and I thought that we also need to authenticate this part). For your suggestion, I must admit that I did not think about `NoWaitStrategy`; totally forgot about it...